components/i2c_output/oled128x32default/definition.json

```diff
@@ -0,0 +1,13 @@
+{
+  "displayName": "128x32 OLED (Default)",
+  "published": true,
+  "vendor": "Adafruit",
+  "productURL": "https://www.adafruit.com/product/938",
```

Comment:
That PID is 1.3" 128x64, should be 4440 like beneath

Reply:
also address is wrong, should be 0x3c